README.md

```diff
@@ -1,24 +1,272 @@
-# Requirements
+Corva python-sdk is a framework for building stream, scheduled and task apps.
+
+## Contents
+
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [App types](#app-types)
+- [Event](#event)
+- [Api](#api)
+- [Cache](#cache)
+- [Contributing](#contributing)
+
+## Requirements
+
 Python 3.8+
```

Comment:
is there any chance that it will work on 3.7 too?

Reply:
3.7 is still widely used. We don't know which version did user used when written his algorithms. How hard it will be to check if everything works for python 3.7?

Reply:
We can potentially support Python 3.6, 3.7, and 3.8, as those are the available AWS Lambda runtimes. We probably should do at least 3.7 though, since 3.6 will be EOL at the end of 2021.

Reply:
Created a task to add 3.7 support [link](https://corvaqa.atlassian.net/browse/DC-869)